src/Orchard.Web/Modules/Orchard.MediaLibrary/Scripts/media-library-picker.js

```diff
@@ -14,6 +14,7 @@
         var promptOnNavigate = element.data("prompt-on-navigate");
         var showSaveWarning = element.data("show-save-warning");
         var addButton = element.find(".button.add");
+        var saveButton = $('.button.save');
```

Comment:
It would be better if the media library picker triggers an event when its list of media ids changes. Then some external script can listen for that event and toggle the Save button's visibility accordingly.